scripting/get5/pausing.sp

```diff
@@ -2,6 +2,38 @@ public bool Pauseable() {
   return g_GameState >= Get5State_KnifeRound && g_PausingEnabledCvar.BoolValue;
 }
 
+public bool PauseGame(MatchTeam team, PauseType type) {
+
+    Get5MatchPausedEvent event = new Get5MatchPausedEvent(g_MatchID, team, type);
+
+    LogDebug("Calling Get5_OnMatchPaused()");
+
+    Call_StartForward(g_OnMatchPaused);
+    Call_PushCell(event);
+    Call_Finish();
+
+    EventLogger_LogAndDeleteEvent(event);
+
+    return Pause(g_FixedPauseTimeCvar.IntValue, MatchTeamToCSTeam(team));
```

Comment:
This needs to be changed I think - if a user has a set tactical pause time but unlimited tech pause time, will this still hit? We had a user in Discord report that their technical pauses are still being fired off with timers. May need an if statement to check for what kind of pause it is and if those values are non-zero?